<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encaminhamento</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icone/ajudante_icone.ico</normaloff>../icone/ajudante_icone.ico</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>Encaminhamento</string>
   </property>
   <widget class="QLineEdit" name="input_problema">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Resumo do problema</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_gerar_nota_encaminhamento">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gerar Nota</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fila de Atendimento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Procedimentos realizados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Diagnóstico:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resumo do problema:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_situacao_mensagem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Quando ocorre (se houver)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_tipo_nota_encaminhamento">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Equipe Caixa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Equipe Externo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Equipe Incidentes</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="input_diagnostico">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>O que está ocorrendo?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_mensage_erro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Descreva mensagem de erro (se houver)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Motivo do encaminhamento:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_motivo_encaminhamento">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Porque está encaminhando a este setor?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mensagem de erro:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_procedimento_realizado">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>O que o técnico fez?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_nao_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>NÃO</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Em que situação ocorre mensagem de erro:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tipo de Nota:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tem imagem em anexo?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_sim_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SIM</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_fila_encaminhamento">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>2000</width>
      <height>2000</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Ambiente Windows</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Automação Bancária e SIGAT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cadastramento e Senha</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Comunicação Eletronica</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Conformidade</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Impreossoras e Relatórios VDR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Instalação de Software</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sistemas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suporte a Navegadores</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
